packages/@o3r/components/builders/component-extractor/index.ts

```diff
@@ -151,7 +151,6 @@ export default createBuilder(createBuilderWithMetricsIfInstalled<ComponentExtrac
    * @param libraries Libraries to watch
    */
   const watchFiles = (libraries: string[]): chokidar.FSWatcher => {
-    // TODO find a better way to watch files
```

Comment:
Did we find a better way to watch those files ?

Reply:
Actually we are using chokibar  to watch the files, not sure about what we called a "better way" so I removed the comment.
If you find what be done in "better way" I am OK to re-put it with more explicit explanation